jina/orchestrate/pods/container.py

```diff
@@ -290,7 +289,7 @@ def __init__(self, args: 'argparse.Namespace'):
             self.args.docker_kwargs.pop('extra_hosts')
         self._net_mode = None
         self.worker = None
-        self.daemon = self.args.daemon  #: required here to set process/thread daemon
+        self.daemon = True  #: required here to set process/thread daemon
```

Comment:
this should be removed.

Reply:
This should go to the `worker` itself

Reply:
I probably messed up when juggling between the two PR